NitroxModel/Packets/CoffeeMachineUse.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using NitroxModel.DataStructures;
+
+namespace NitroxModel.Packets;
+
+[Serializable]
+public sealed class CoffeeMachineUse : Packet
+{
+    public NitroxId Id { get; }
+    public CoffeeMachineSlot Slot { get; }
+
+    public CoffeeMachineUse(NitroxId id, CoffeeMachineSlot slot)
+    {
+        Id = id;
+        Slot = slot;
+    }
+}
+
+public enum CoffeeMachineSlot
+{
+    ONE,
+    TWO
+}
```

Comment:
That way it's not exposed to every class having `using NitroxModel.Packets`
```suggestion

  public enum CoffeeMachineSlot
  {
      ONE,
      TWO
  }
}

```